@here/harp-examples/src/threejs_add-animated-object.ts

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Licensed under Apache 2.0, see full license in LICENSE
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { HelloWorldExample } from "@here/harp-examples/src/hello";
+import { GeoCoordinates } from "@here/harp-geoutils";
+import { MapViewEventNames, MapViewUtils, RenderEvent } from "@here/harp-mapview";
+import * as THREE from "three";
+
+// tslint:disable-next-line:no-var-requires
+(window as any).Zlib = require("three/examples/js/libs/inflate.min.js").Zlib;
+
+import "three/examples/js/loaders/FBXLoader";
+
+/**
+ * This example builds on top of the [[HelloWorldExample]], so please consult that first for any
+ * questions regarding basic setup of the map.
+ *
+ * Additionaly this example shows how to add a animated [THREE.js](https://threejs.org/) object
+ * to the scene.
+ *
+ * Therefore the object is loaded with one of the [THREE.js](https://threejs.org/) loaders (we chose
+ * FBX here).
+ * ```typescript
+ * [[include:harp_gl_threejs_add_animated-object_load.ts]]
+ * ```
+ *
+ * Once the object is loaded, we add it to the [[MapView.scene]] :
+ * ```typescript
+ * [[include:harp_gl_threejs_add_animated-object_add_to_scene.ts]]
+ * ```
+ *
+ * For ensuring good precision close to the camera harp.gl uses [relative to
```

Comment:
this is described already in the threejs_add-simple-object, I think it would be good to reference that from here (otherwise, we have two descriptions to maintain).
and if that description can be improved, by all means, feel free to edit it.

Reply:
done